ethereum/contracts/WrappedToken.sol

```diff
@@ -1,35 +1,19 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.9;
 
-import "@openzeppelin/contracts/token/ERC777/ERC777.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/draft-ERC20Permit.sol";
 
-contract WrappedToken is ERC777, Ownable {
+/// @custom:security-contact contact@snowfork.com
+contract WrappedToken is ERC20, Ownable, ERC20Permit {
```

Comment:
WrappedToken is just a Vault for dot-app and my assumption is that it will be changed to a vault in the upgradability branch?

Reply:
WrappedToken already acts like a vault. So in production we can transfer ownership of it to a new DOTApp.